src/message/basicMessage.js

```diff
@@ -91,7 +92,7 @@ class BasicMessage{
 
     /**
      * Sets the plain text portion of the message body. 
-     * (Optional) Either textBody or HtmlBody must be used or use a ApiTemplate
+     * (Optional) Either textBody or HtmlBody must be used or use a ApiTemplate with an AmpBody
```

Comment:
rephrase this line: "Either TextBody or HtmlBody must be used with the AmpBody or use a ApiTemplate"